addons/editor/functions/fnc_pingCurators.sqf

```diff
@@ -0,0 +1,101 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand, mharis001
+ * Pings curators to the given target object or position.
+ *
+ * Arguments:
+ * 0: Curator <OBJECT>
+ * 1: Ping Target <OBJECT|ARRAY>
+ *   - Positions must be in AGL format.
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_curator, _target] call zen_editor_fnc_pingCurators
+ *
+ * Public: No
+ */
+
+#define PING_DRAW_TIME 1.2
+#define PING_DRAW_ICON "\a3\ui_f_curator\data\logos\arma3_curator_eye_512_ca.paa"
```

Comment:
Capitalization just in case:
```suggestion
#define PING_DRAW_ICON "\a3\ui_f_curator\Data\Logos\arma3_curator_eye_512_ca.paa"
```

Reply:
Does not matter and we generally use full lowercase path (from unpacked P-drive).

Reply:
Doesn't matter even on Linux ?

Reply:
This would be a path in Arma's virtual file system.

Reply:
Right. Good to know.